sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterImpl.java

```diff
@@ -436,6 +445,31 @@ private void clearKubeConfig() {
         this.formatUserKubeConfigsMap.clear();
     }
 
+    @Override
+    public KubernetesClusterImpl withFreeSku() {
+        this.innerModel().withSku(new ManagedClusterSku().withTier(ManagedClusterSkuTier.FREE).withName(ManagedClusterSkuName.BASE));
+        if (KubernetesSupportPlan.KUBERNETES_OFFICIAL.equals(this.innerModel().supportPlan())) {
+            this.innerModel().withSupportPlan(KubernetesSupportPlan.KUBERNETES_OFFICIAL);
+        }
```

Comment:
What's this? Isn't it same?

Reply:
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/db0171ac-3ab9-44da-99cf-5c821edc8f6f)

Reply:
Fixed in the new version.